packer/playbook-bastion.yaml

```diff
@@ -0,0 +1,7 @@
+- name: Installing furyagent
+  hosts: all
+  become: true
+  vars:
+    furyagent_uri: https://github.com/sighupio/furyagent/releases/download/v0.0.5/furyagent-linux-amd64
+  roles:
+    - ../../roles/furyagent
```

Comment:
We should also install the openvpn binaries, this would require the openvpn role to be refactored and probably split into installation and configuration. This is not blocking, but would be nice.

Reply:
Annotated in our internal trello board.